RecoEcal/EgammaClusterProducers/python/SCEnergyCorrectorDRNProducer_cfi.py

```diff
@@ -0,0 +1,46 @@
+import FWCore.ParameterSet.Config as cms
+
+DRNProducerEB = cms.EDProducer('SCEnergyCorrectorDRNProducer',
+   correctorCfg = cms.PSet(
+     ecalRecHitsEE = cms.InputTag('reducedEcalRecHitsEE'),
+     ecalRecHitsEB = cms.InputTag('reducedEcalRecHitsEB'),
+     rhoFastJet = cms.InputTag("fixedGridRhoAll"),
+   ),
+   inputSCs = cms.InputTag('particleFlowSuperClusterECAL','particleFlowSuperClusterECALBarrel'),
+   Client = cms.PSet(
+        mode = cms.string("Async"),
+        preferredServer = cms.untracked.string(""),
+        timeout = cms.untracked.uint32(10),
+        modelName = cms.string("MustacheEB"),
+        modelVersion = cms.string(""),
+        modelConfigPath = cms.FileInPath("RecoEcal/EgammaClusterProducers/data/models/MustacheEB/config.pbtxt"),
+        verbose = cms.untracked.bool(False),
+        allowedTries = cms.untracked.uint32(1),
+        useSharedMemory = cms.untracked.bool(True),
+        compression = cms.untracked.string(""),
+    ),
+ )
+
+
+DRNProducerEE = cms.EDProducer('SCEnergyCorrectorDRNProducer',
+   correctorCfg = cms.PSet(
+     ecalRecHitsEE = cms.InputTag('reducedEcalRecHitsEE'),
+     ecalRecHitsEB = cms.InputTag('reducedEcalRecHitsEB'),
+     rhoFastJet = cms.InputTag("fixedGridRhoAll"),
+   ),
+   inputSCs = cms.InputTag('particleFlowSuperClusterECAL','particleFlowSuperClusterECALEndcapWithPreshower'),
+   Client = cms.PSet(
+        mode = cms.string("Async"),
```

Comment:
same comment as above regarding Triton-specific technical configurations.

Reply:
Yes, the fillDescription default can just be used here.

Reply:
based on the above discussion, these should come from the fillDescriptions, rather than being specified explicitly in the python file (marked as resolved but not really resolved AFAICT).

Reply:
Can you clarify which configuration parameters should come from the fillDescriptions here? The inputSCs must be specified in the python file as they are different for EB and EE, and I am not sure which of the Client parameters are pre-specified by Triton and what their defaults are.

Reply:
SONIC provides fillDescriptions for all Client-related parameters:
https://github.com/cms-sw/cmssw/blob/8b0ba7c1bf63e1fd54530c292df0ddd4d00c4236/HeterogeneousCore/SonicCore/src/SonicClientBase.cc#L73
https://github.com/cms-sw/cmssw/blob/8b0ba7c1bf63e1fd54530c292df0ddd4d00c4236/HeterogeneousCore/SonicTriton/src/TritonClient.cc#L406

The "PseudoAsync" default mode is somewhat of a historical setting; "Async" is the current recommendation for SonicTriton.

Reply:
@ssrothman can you implement the specification of the client parameters from the ones provided by fillDescriptions (rather than explicitly specifying them again as is currently done)?

Reply:
Yes, will do.